Code/Tools/Libs/GuiFoundation/PropertyGrid/Implementation/PropertyBaseWidget.cpp

```diff
@@ -406,7 +406,7 @@ ezVariant ezQtPropertyWidget::GetCommonValue(const ezHybridArray<ezPropertySelec
     {
       if (!value.IsValid())
       {
-        m_pObjectAccessor->GetValue(item.m_pObject, pProperty, value, item.m_Index).AssertSuccess();
+        m_pObjectAccessor->GetValue(item.m_pObject, pProperty, value, item.m_Index).IgnoreResult();
```

Comment:
Does the rest of the code handle the failure case properly?

Reply:
Don't know, I was running into this assert when changing the script resource property on a script component to one that had different exposed parameters, so GetValue failed trying to retrieve the old values.